css/03-generic.css

```diff
@@ -119,6 +119,12 @@ h4 {
 	transform: rotate(-2deg);
 }
 
+/* Remove H1 from home */
+
+body.home header.entry-header {
+	display: none;
```

Comment:
It is harmful for accessibility / SEO to use display: none to hide elements ([more about that here](https://css-tricks.com/places-its-tempting-to-use-display-none-but-dont/)) - use these declarations instead:
```
position:absolute;
left: -10000px;
top: auto;
width: 1px;
height: 1px;
overflow: hidden;
```